### ISSUE TITLE
Current stable version?

### ISSUE THREAD

**Issue by magicwerk** [javaparser/javaparser#3719]
I have a hard time figuring out what is the latest version stable version to use:

Main page https://github.com/javaparser/javaparser tells:
```
<dependency>
    <groupId>com.github.javaparser</groupId>
    <artifactId>javaparser-symbol-solver-core</artifactId>
    <version>3.24.6</version>
</dependency>
```

However the lastest version found in MVN repo is
https://mvnrepository.com/artifact/com.github.javaparser/javaparser-core
3.24.4

Tags mentions:
https://github.com/javaparser/javaparser/tags
3.24.6 (from 26 Aug)

But releases (also on main page) tells you 3.24.3 is the latest one:
https://github.com/javaparser/javaparser/releases
3.24.3 

So which is the lastest version to use?
Any chance to clean this up?
This seems somehow to be related to #2968

**Comment by jlerbsc** [comment]
Hi @magicwerk i understand your dismay! It's my fault. I tried to understand and follow the procedure described by @MysterAitch  but I encountered a problem on the preparation of the release. I didn't realize that early enough. I will try to put everything back in order during the next release. Thanks for understanding.
In the meantime the version which must be considered stable is the one published on maven (3.24.4)
